app_data/sheets/data_list/generated/create_article_tasks.json

```diff
@@ -519,6 +520,7 @@
         }
       },
       "media_type": "audio",
+      "media_asset": "images/modules/create/create_03.mp2",
```

Comment:
@30ann is it correct that some of the audio assets are `.mp2` files or is that a typo?